openshift_cli_installer/utils/cli_utils.py

```diff
@@ -533,3 +565,91 @@ def prepare_clusters(clusters, ocm_token):
             )
 
     return clusters
+
+
+def click_echo(name, platform, section, msg, success=None, error=None):
+    if success:
+        fg = SUCCESS_LOG_COLOR
+    elif error:
+        fg = ERROR_LOG_COLOR
+    else:
+        fg = "white"
+
+    click.secho(
+        f"[Cluster: {name} - Platform: {platform} - Section: {section}]: {msg}", fg=fg
+    )
+
+
+def assert_managed_acm_clusters_user_input(clusters, create):
+    if create:
+        section = "Verify user input"
+        for cluster in clusters:
+            managed_acm_clusters = get_managed_acm_clusters_from_user_input(
+                cluster=cluster
+            )
+            for managed_acm_cluster in managed_acm_clusters:
+                managed_acm_cluster_data = get_cluster_data_by_name_from_clusters(
+                    name=managed_acm_cluster, clusters=clusters
+                )
+                if not managed_acm_cluster_data:
+                    click_echo(
+                        name=managed_acm_cluster,
+                        platform=None,
+                        section=section,
+                        error=True,
+                        msg=f"Cluster {managed_acm_cluster} not found",
+                    )
+                    raise click.Abort()
+                managed_acm_cluster_platform = managed_acm_cluster_data["platform"]
+                if managed_acm_cluster_platform not in SUPPORTED_PLATFORMS:
+                    click_echo(
+                        name=managed_acm_cluster_platform["name"],
+                        platform=managed_acm_cluster_platform,
+                        section=section,
+                        error=True,
+                        msg=f"{managed_acm_cluster_platform} is not supported",
+                    )
+                    raise click.Abort()
```

Comment:
do we need this? once we check that the cluster exists and we also check supoprted platforms then this can be removed, right?